components/Marker/index.jsx

```diff
@@ -6,11 +6,16 @@ import { Icon } from 'leaflet';
 import styles from './style.module.css';
 
 const getIcon = (type) => {
-  if (type === 'golden') {
-    return new Icon({ iconUrl: '/logo-golden.png', iconSize: [30, 30] });
+  switch (type) {
+    case 'sticker':
+      return new Icon({ iconUrl: '/marker_sticker.png', iconSize: [40, 80] });
+    case 'picture':
+      return new Icon({ iconUrl: '/marker_picture.png', iconSize: [40, 80] });
+    case 'special':
+      return new Icon({ iconUrl: '/marker_special.png', iconSize: [40, 80] });
+    default:
+      return new Icon({ iconUrl: '/marker_sticker.png', iconSize: [40, 80] });
```

Comment:
Super cool! I think it would be cool to have a folder `/public/images/markers/{type}.png` to have it more organized.

Reply:
Done!